packages/styles/stories/BTP/Navigation/horizontal/navigation-horizontal-expander.example.html

```diff
@@ -1,19 +1,18 @@
 <div style="height: 500px">
     <nav class="fd-navigation fd-navigation--horizontal" aria-label="Horizontal Navigation">
         <div class="fd-navigation__container">
-            <ul class="fd-navigation__list" role="menubar" aria-label="Horizontal Navigation">
+            <ul class="fd-navigation__list" role="tablist" aria-orientation="horizontal" aria-label="Horizontal Navigation">
                 <li
                     role="none"
                     tabindex="-1"
                     class="fd-navigation__list-item fd-navigation__list-item--home"
-                    aria-hidden="true"
                 >
                     <div
                         class="fd-navigation__item"
                         aria-current="page"
                         aria-selected="false"
                         aria-expanded="false"
-                        role="menuitem"
+                        role="tab"
                     >
                         <a class="fd-navigation__link" role="link" href="#">
```

Comment:
in this case, since we now have Tabs, I am not sure if we need the role link and the href

Reply:
Well, technically it's still a navigation, and link can be applied to it, no?